batch-nft-reveal/app/.eslintrc.json

```diff
@@ -10,6 +10,7 @@
     // Is not enabled right now to avoid issues with the Next.js repo
     // "prettier",
   ],
+  "root": true,
```

Comment:
How is this fixing the problem and what was the problem?

Reply:
The problem was that ESLint looks for eslint file recursively  and there are two eslintrc in our repo and it can't determine which one to use so there's a conflict. The first one is in batch-nft-reveal/.eslintrc.js and the other one is in batch-nft-reveal/app/eslintrc.json. By setting `root:true` it tells the eslint to not look for more parent folders.